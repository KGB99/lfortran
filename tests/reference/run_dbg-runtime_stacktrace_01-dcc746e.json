{
    "basename": "run_dbg-runtime_stacktrace_01-dcc746e",
    "cmd": "lfortran {infile} -g --debug-with-line-column --no-color",
    "infile": "tests/errors/runtime_stacktrace_01.f90",
    "infile_hash": "3e65758e10744eea8d101be6437af3724c0ec32c832e29da968fc22d",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "run_dbg-runtime_stacktrace_01-dcc746e.stdout",
    "stdout_hash": "3bb54d94e2d111c5ecb9eb370562d40ff2806b07b914f3f3fa45f3ce",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 6
}