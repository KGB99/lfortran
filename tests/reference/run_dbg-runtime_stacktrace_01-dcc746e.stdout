The stack address was not found in any shared library or the main program, the stack is probably corrupted.
Aborting...
